components/Banner/examples/supplemental/MDCBannerView.m

```diff
@@ -14,29 +14,42 @@
 
 #import "MDCBannerView.h"
 
+#import "MDCBannerViewConstants.h"
 #import "MDCBannerViewLayout.h"
 #import "MaterialButtons.h"
+#import "MaterialTypography.h"
 
-static const CGFloat kIconImageContainerSideLength = 40.0f;
-static const CGFloat kIconImageSideLength = 24.0f;
-static const CGFloat kTextColorOpacity = 0.87f;
-static const CGFloat kTextFontSize = 14.0f;
+static const CGFloat kIconImageViewSideLength = 40;
 static const NSInteger kTextNumberOfLineLimit = 3;
+#if DEBUG
 static const NSUInteger kNumberOfButtonsLimit = 2;
+#endif
+
+static const uint32_t MDCBannerDefaultBackgroundColor = 0xFFFFFFFF;
```

Comment:
Tip: `UIColor.whiteColor`

Reply:
Addressed in https://github.com/material-components/material-components-ios/pull/6830/commits/3a9c5afbbf966c8a2ef5a6e8ad5d76641534b057#diff-9f546e78faa17de3d67395c6ef3629b2R61.